client/layout/guided-tours/tours/media-basics-tour.js

```diff
@@ -99,9 +104,9 @@ export const MediaBasicsTour = makeTour(
 			style={ { marginLeft: '-8px' } }
 		>
 			<Continue click step="launch-modal" target=".editor-media-modal__secondary-action">
-				{ translate( 'Now click the {{strong}}Edit{{/strong}} button.', {
+				{ translate( 'Now click the {{editButton/}} button.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Now click the {{EditButton/}} button.' )` *ES Score: 20*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).